metadata-ingestion/src/datahub/ingestion/source/dremio/dremio_sql_queries.py

```diff
@@ -251,7 +251,7 @@ class DremioSQLQueries:
         SELECT
             *
         FROM
-            SYS.PROJECT.HISTORY.JOBS
+            sys.project.history.jobs
```

Comment:
Is this case change necessary ? Is it also required for other sql queries from this file ?

Reply:
@dremio-brock has been doing some testing and found that this query specifically was giving a null pointer exception with Dremio unless this was changed to lowercase. This is a workaround to a Dremio Cloud bug.